index.html

```diff
@@ -8,21 +8,45 @@
     <title>Awesome BookList</title>
 </head>
 <body>
+    <div class="head">
+        <p>Awesome books</p>
+        <nav>
+            <ul class="nav">
+                <li class="nav-item1"><a href="#table-section">List</a></li>
+                <li><hr></li>
+                <li class="nav-item2"><a href="#addbook">Add new</a></li>
+                <li><hr></li>
+                <li class="nav-item3"><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+    </div>
+    <span id='date-time'></span>
+    <div id="table-section" class="table-section">
     <h1>All Awesome Books</h1>
-    <table class="book-list">
-    </table>
-    <form id="form" class="form1">
-        <h1>Add a new book</h1>
-        <div>
-            <!-- <label >Title</label><br> -->
-            <input type="text" id="title" class="title1" placeholder="title" required>
-        </div>
-        <div>
-            <!-- <label >Author</label><br> -->
-            <input type="text" id="author" class="author1" placeholder="author" required>
-        </div>
-        <button type="submit" class="add">Add</button>
-    </form>
+    <table class="book-list"></table>
+    </div>
+    <div id="addbook" class="addbook">
+        <form id="form" class="form1">
+            <h1>Add a new book</h1>
+            <div>
+                <input type="text" id="title" class="title1" placeholder="title" required>
+             </div>
+            <div>
+                <input type="text" id="author" class="author1" placeholder="author" required>
+            </div>
+            <button type="submit" class="add">Add</button>
+        </form>
+    </div>
+    <div id="contact" class="contact">
+        <h1>Contact Information</h1>
+        <p>Do you have any questions or just want to say "Hello"?<br>You can reach out to us!</p>
+        <ul>    
+            <li>Our email: ericandsharon@gmail.com</li>
+            <li>Our phone number: 012345678 </li>
+            <li>Our address: Kilimani, 80, 0100 Nairobi, Kenya </li>
+        </ul>
+    </div>
+    <footer>Copyright ...</footer>
 <script src="scripts/index.js"></script>
```

Comment:
- [ ] Kindly make the footer always stay at the bottom of the page even when there is no content or books on the page. Take a look at the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#project-requirements) and see that the footer is at the bottom of the page. Here is a [link](https://dev.to/nehalahmadkhan/how-to-make-footer-stick-to-bottom-of-web-page-3i14) to see how to make the footer to stay at the bottom of the page.

- [ ] Also make the width of your page to be bigger so that it covers most of the body of the page. Check the wireframe to see the difference.